app/views/admin/creator_settings/_form.html.erb

```diff
@@ -8,17 +8,14 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <%= label_tag :logo_svg, class: "crayons-field__label" do %>
+  <%= label_tag :logo, class: "crayons-field__label" do %>
```

Comment:
Renamed the `name` attribute of the logo upload input to `logo`.

Reply:
We need to deal with current and previous cases where logo_svg exists as a setting and we have the contents of an svg logo saved to the database. Maybe we can create some data update script, or perhaps we can add the `logo` field and keep `logo_svg` as well and then phase out logo_svg as a separate PR. It seems safer that way.

Reply:
This is just the form input's name being changed. The only change on the backend at the moment is that the allowed params now has a `logo` param instead of `logo_svg`.